### ISSUE TITLE
Wallet Overview - add additional buckets for claims/tips/supports

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#617]
## The Issue
Currently, a user is only able to see an LBC balance which denotes total available LBC in the wallet.  Would be a nice feature to show additional information/breakdowns for tips and locked up LBC via supports.  This will also make users more aware of where their LBC is allocated, so there are fewer questions after reviewing their transaction list - they can add up the balances. 

Ideas:
Total LBC locked up in claims (channel claims + publish claims)
Total LBC Supporting my claims( breakdown with own supports vs others' supports)
Total LBC Supporting other claims
Total LBC in Tips (include self tips, or include them as supports instead?)

Above is up for discussion/direction. 

### Expected behaviour
Overview would show balance of locked up tips, own supports, others' supports and total locked up in claims/supports 

### Actual behaviour
We only have wallet balance at this time

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by eggplantbren** [comment]
I was just going to open an issue for this exact thing. A simple start could be to have something like "LBC in wallet" and "LBC net worth", where the latter includes supports and LBC held against claims. I imagine it would be quite easy to both compute that and display it in the app.

**Comment by tzarebczan** [comment]
This is now possible with 0.38.7, which has a breaking API change for account balance: 

{
  "total": "all spendable in my wallet, including claims/supports",
  "reserved": "tied up in claims/supports",
  "available": "total - reserved",
  "reserved_subtotals": {
    "claims": "in active claims",
    "supports": "in active supports",
    "tips": "in active tips"
  }
}

Just account balance, which should run just as fast as before (did notice some slowness on Windows with super large wallet, so we may want to refresh every 4 seconds instead of 2 until we get web sockets). This would be run in the background to keep available balance updated. Maybe we can even show reserved total as a hover? 
```
{
  "available": "251.1298971",
  "reserved": "991.471026",
  "reserved_subtotals": null,
  "total": "2493.7308202"
}
```
with --reserved_subtotals (would be run when on wallet overview page)

```
{
  "available": "251.1298971",
  "reserved": "991.471026",
  "reserved_subtotals": {
    "claims": "49.471026",
    "supports": "822.0",
    "tips": "120.0"
  },
  "total": "2493.7308202"
}
```

**Comment by tzarebczan** [comment]
Notes from planning:
The main balance we show should be the Total (does this include claims? Maybe not since we don't want them unlocking it). 

Then we would show total available for sending, total locked in claims, total unlockable tips, total locked in supports (should this be with tips)? 

There's should be an unlock all button which would abandon any tips/supports (or do these separately?) 

When trying to send a value above the available balance, mention that X are locked in tips/supports. with at least an indication on how to unlock.
